### PR TITLE
UIORGS-257 Organizations - Implement MCL Next/Previous pagination

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://issues.folio.org/browse/UIORGS-257
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/common/resources/organization.js [hunk 1]
```diff
@@ -28,4 +28,4 @@ export const fetchOrgsByParam = {
   fetch: false,
 };
 
-export const RESULT_COUNT_INCREMENT = 30;
+export const RESULT_COUNT_INCREMENT = 100;
```
Comment (NikitaSedyx) [inline src/common/resources/organization.js:31]: why 100?

Comment (NikitaSedyx) [reply src/common/resources/organization.js:31]: I see from requirements, need to discuss with Dennis, 100 can be expensive on some screens

Comment (usavkov-epam) [reply src/common/resources/organization.js:31]: Ok)

File: src/Organizations/OrganizationsList/hooks/useOrganizations/useOrganizations.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    if (!filtersCount) {
+      return { organizations: [], totalRecords: 0 };
+    }
+
+    return ky
+      .get(VENDORS_API, { searchParams: { ...defaultSearchParams, ...searchParams } })
+      .json()
+      .then(res => {
+        if (res.totalRecords !== null) {
+          resultsPaneTitleRef?.current?.focus();
```
Comment (NikitaSedyx) [inline src/Organizations/OrganizationsList/hooks/useOrganizations/useOrganizations.js:49]: why is it in hook?

Comment (NikitaSedyx) [reply src/Organizations/OrganizationsList/hooks/useOrganizations/useOrganizations.js:49]: in hooks with responsibility to fetch data

Comment (usavkov-epam) [reply src/Organizations/OrganizationsList/hooks/useOrganizations/useOrganizations.js]: removed it

File: src/Organizations/OrganizationsList/hooks/useBuildQuery/useBuildQuery.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import {
+  getKeywordQuery,
+} from '../../OrganizationsListSearchConfig';
+import {
+  customFilterMap,
+  CUSTOM_SORT_MAP,
+} from '../../OrganizationsListFilter/OrganizationsListFilterConfig';
+
+export const useBuildQuery = () => {
+  const buildQuery = useCallback(makeQueryBuilder(
```
Comment (NikitaSedyx) [inline src/Organizations/OrganizationsList/hooks/useBuildQuery/useBuildQuery.js:16]: return useCallback

currently it's code smell

**Review by NikitaSedyx — APPROVED** [review]
the rest is lgtm except code smell and 100 limit

**Review by AndreiShumski2 — APPROVED** [review]

File: src/Organizations/OrganizationsList/hooks/useBuildQuery/useBuildQuery.js (hunk 3 above)
Comment (usavkov-epam) [reply src/Organizations/OrganizationsList/hooks/useBuildQuery/useBuildQuery.js]: updated

**Pull request merged**